gitlab_templates/force_build_repo_image.yml

```diff
@@ -0,0 +1,20 @@
+stages:
+  - build
+
+# The only difference between this file and the build_repo_image.yml is that
+# it does not have a rules section
```

Comment:
You only included this comment in some of the files, I don't know if that was intentional or not.